<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImplantDesignClass</class>
 <widget class="QMainWindow" name="ImplantDesignClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImplantDesign</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ImplantDesign.qrc">
    <normaloff>:/ImplantDesign/Resources/smartguideicon.ico</normaloff>:/ImplantDesign/Resources/smartguideicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QCustomVTKWidget" name="ViewWidget" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="ParameterWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Model </string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QTableWidget" name="ModelTableWidget">
            <property name="maximumSize">
             <size>
              <width>330</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Parameter</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="3" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Thickness:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="ThicknessSpinBox">
              <property name="minimum">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>X Extend:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="XExtendSpinBox">
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>300</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Y Extend:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="YExtendSpinBox">
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>300</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Z Extend:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="ZExtendSpinBox">
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>300</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="ContourChooseButton">
              <property name="styleSheet">
               <string notr="true">QPushButton:checked { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);

	                 font: 75 16pt &quot;Arial&quot;;
					
					
					color: rgb(255, 78, 34);
                    }</string>
              </property>
              <property name="text">
               <string>Contour Choose</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="SingleChooseButton">
              <property name="styleSheet">
               <string notr="true">QPushButton:checked { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);

	                 font: 75 16pt &quot;Arial&quot;;
					
					
					color: rgb(255, 78, 34);
                    }</string>
              </property>
              <property name="text">
               <string>Single Choose</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>215</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadProject"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadProject">
   <property name="icon">
    <iconset theme="LoadProject" resource="ImplantDesign.qrc">
     <normaloff>:/ImplantDesign/Resources/openproject.ico</normaloff>:/ImplantDesign/Resources/openproject.ico</iconset>
   </property>
   <property name="text">
    <string>LoadProject</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomVTKWidget</class>
   <extends>QWidget</extends>
   <header>QCustomVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ImplantDesign.qrc"/>
 </resources>
 <connections/>
</ui>
